### DISCUSSION TITLE
Is it possible to change the color of the checkbox?

### DISCUSSION THREAD

**Discussion by kocur4d** [mantinedev/mantine#1191] [Q&A] [answered]
I am using some checkboxes in my forms but I am not using mantine form.

Other inputs have prop exposed `invalid` which works perfectly fine.

Is there a way to achieve something similar with a checkbox?

<img width="381" alt="Screenshot 2022-04-06 at 16 50 57" src="https://user-images.githubusercontent.com/1427354/162015975-4fee811c-7967-4dce-b8f6-1483434efc18.png">

I would just like to change the color of the highlighted square to red similar how phone number input has its border set to red.

Cheers

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/patient-lake-5z6cf4?file=/src/App.tsx

**Reply by kocur4d** [reply]
Cheers that worked.

Is there any way to accomplish similar, overwrite the `input` styles, by using `createStyles` API?

I am using colors from a theme and to extract them to use in `styles` prop is not very elegant.

**Reply by rtivital** [reply]
https://mantine.dev/theming/styles-api/

**Reply by kocur4d** [reply]
OK I think I got it. Thank you.

I could do something like this

```
const useStyles = createStyles((theme) => ({
  checkbox: {
    input: {
      border: "1px solid red",
    },
  }
}))
```
and apply `checkbox` class to the `Checkbox` conditionally.

**Comment by kocur4d** [comment]
Ok so by passing `styles` to `Checkbox` I can update props but wouldn't it be nicer to have similar API like other `Input`'s via some prop like `invalid`?
